gpkit/interactive/widget.py

```diff
@@ -30,17 +30,17 @@ def outputfn(gp):
             print gp.solution.table(["cost", "free_variables"])
 
     gp.sweep = {}
-    gp.prewidget = gp.last
+    # gp.prewidget = gp.last
 
-    solveargs["printing"] = False
+    solveargs["verbosity"] = 0
     def display(**subs):
-        gp.sub(subs, replace=True)
-        if hasattr(gp, "localsolve"):
-            gp.localsolve(**solveargs)
-        else:
-            gp.solve(**solveargs)
+        gp.substitutions.update(subs)
+        # if hasattr(gp, "localsolve"):
+        #     gp.localsolve(**solveargs)
+        # else:
+        gp.solve(**solveargs)
```

Comment:
No actually, I think `gp` is a `Model` here (I wanted to minimize changes to the code to get things working, so I retained the variable name `gp`, but this is clearly bad naming). 

`Model` does not have a `sub` method, so I had to do the hack above instead.